v1.10.1 assert errors

It appears that the 1.10.1 release contains several [assert functions](https://github.com/scssphp/scssphp/blob/v1.10.1/src/Formatter.php#L352) that is causing `Uncaught AssertionError: assert($this->currentBlock->sourceLine !== null)`. This appears to have already changed on `master` so I'm guessing it got overlooked when the release was created?

--- a/src/Compiler.php
+++ b/src/Compiler.php
@@ -722,9 +722,9 @@ class Compiler
             $out->sourceLine   = $this->env->block->sourceLine;
             $out->sourceColumn = $this->env->block->sourceColumn;
         } else {
-            $out->sourceName   = null;
-            $out->sourceLine   = null;
-            $out->sourceColumn = null;
+            $out->sourceName = isset($this->sourceNames[$this->sourceIndex]) ? $this->sourceNames[$this->sourceIndex] : '(stdin)';
+            $out->sourceLine = $this->sourceLine;
+            $out->sourceColumn = $this->sourceColumn;
         }
 
         return $out;

--- a/src/Parser.php
+++ b/src/Parser.php
@@ -1638,19 +1638,19 @@ class Parser
 
                 if (! $comment) {
                     // single part static comment
-                    $this->appendComment([Type::T_COMMENT, $c]);
+                    $commentStatement = [Type::T_COMMENT, $c];
                 } else {
                     $comment[] = $c;
                     $staticComment = substr($this->buffer, $startCommentCount, $endCommentCount - $startCommentCount);
                     $commentStatement = [Type::T_COMMENT, $staticComment, [Type::T_STRING, '', $comment]];
-
-                    list($line, $column) = $this->getSourcePosition($startCommentCount);
-                    $commentStatement[self::SOURCE_LINE] = $line;
-                    $commentStatement[self::SOURCE_COLUMN] = $column;
-                    $commentStatement[self::SOURCE_INDEX] = $this->sourceIndex;
-
-                    $this->appendComment($commentStatement);
                 }
+
+                list($line, $column) = $this->getSourcePosition($startCommentCount);
+                $commentStatement[self::SOURCE_LINE] = $line;
+                $commentStatement[self::SOURCE_COLUMN] = $column;
+                $commentStatement[self::SOURCE_INDEX] = $this->sourceIndex;
+
+                $this->appendComment($commentStatement);
 
                 $this->commentsSeen[$startCommentCount] = true;
                 $this->count = $endCommentCount;

--- a/tests/FrameworkTest.php
+++ b/tests/FrameworkTest.php
@@ -25,6 +25,7 @@ class FrameworkTest extends TestCase
     {
         $compiler = new Compiler();
         $compiler->setLogger(new QuietLogger());
+        $compiler->setSourceMap(Compiler::SOURCE_MAP_INLINE);
 
         $entrypoint = dirname(__DIR__) . '/vendor/twbs/bootstrap/scss/bootstrap.scss';
 
@@ -37,6 +38,7 @@ class FrameworkTest extends TestCase
     {
         $compiler = new Compiler();
         $compiler->setLogger(new QuietLogger());
+        $compiler->setSourceMap(Compiler::SOURCE_MAP_INLINE);
 
         $entrypoint = dirname(__DIR__) . '/vendor/twbs/bootstrap4/scss/bootstrap.scss';
 
@@ -50,6 +52,7 @@ class FrameworkTest extends TestCase
         $compiler = new Compiler();
         $compiler->addImportPath(dirname(__DIR__) . '/vendor/twbs/bootstrap4/scss');
         $compiler->setLogger(new QuietLogger());
+        $compiler->setSourceMap(Compiler::SOURCE_MAP_INLINE);
 
         $scss = <<<'SCSS'
 $enable-shadows: true;
@@ -67,6 +70,7 @@ SCSS;
     {
         $compiler = new Compiler();
         $compiler->setLogger(new QuietLogger());
+        $compiler->setSourceMap(Compiler::SOURCE_MAP_INLINE);
 
         $entrypoint = dirname(__DIR__) . '/vendor/zurb/foundation/assets/foundation.scss';
 
@@ -82,6 +86,7 @@ SCSS;
     {
         $compiler = new Compiler();
         $compiler->setLogger(new QuietLogger());
+        $compiler->setSourceMap(Compiler::SOURCE_MAP_INLINE);
         $compiler->addImportPath(dirname(__DIR__) . '/vendor/thoughtbot/bourbon');
         $compiler->addImportPath(dirname(__DIR__) . '/vendor/thoughtbot/bourbon/spec/fixtures');
 

